### ISSUE TITLE
Advert size error

### ISSUE THREAD

**Issue by jamiecarter7** [mazipan/vue-google-adsense#25]
Im having a weird issue where I am using a set size ad unit (200px x 200px).

I have set the wrapper of this advert at height 400px.

I have used the correct data-ad-slot code where I have set this slot only to give 200x200, yet when loading the site im getting an advert 600px height created.

```<ins data-ad-client="REDACTED" data-ad-slot="REDACTED" data-ad-test="" data-ad-format="auto" class="adsbygoogle" style="display: block; height: 600px;" data-adsbygoogle-status="done">```

```<Adsense
            class="adslot-aside"
            :key="rerenderad"
            data-ad-client="ca-pub-5393590966220416"
            data-ad-slot="REDACTED"
          >
          </Adsense>```

```.adslot-aside {
  max-height: 460px;
}```

Any idea why this is happening?

**Comment by jamiecarter7** [comment]
The thing that was throwing me off is the style block inside the ins block

I did some digging and you set the advert type using data-ad-format="auto" [auto, rectangle, vertical, horizontal]

**Comment by mazipan** [comment]
You can create PR if need improvement in this library @jamiecarter7

**Comment by vesper8** [comment]
@jamiecarter7 thanks for your answer.. indeed if you only specify the slot it defaults to the "auto" ad-format and ignores the fixed size you have set on Adsense.. setting 'horizontal' for my leaderboard and 'rectangle' for my rectangles solved the problem.. this should at least be updated in the documentation

**Comment by mazipan** [comment]
Can you add on the docs @vesper8?

**Comment by vesper8** [comment]
@mazipan done: https://github.com/mazipan/vue-google-adsense/pull/60

**Comment by mazipan** [comment]
Released on `1.8.0`
